web/client/components/buttons/ZoomButton.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+const {Button, Glyphicon} = require('react-bootstrap');
+
+const ZoomButton = React.createClass({
+    propTypes: {
+        id: React.PropTypes.string,
+        style: React.PropTypes.object,
+        glyphicon: React.PropTypes.string,
+        text: React.PropTypes.string,
+        btnSize: React.PropTypes.oneOf(['large', 'medium', 'small', 'xsmall']),
+        className: React.PropTypes.string,
+        step: React.PropTypes.number,
+        currentZoom: React.PropTypes.number,
+        onZoom: React.PropTypes.func,
+        tooltip: React.PropTypes.element
```

Comment:
We should implement enable / disable behaviour, so that zoomIn / zoomOut buttons are disabled when reaching min / max zoom